ts/kpt-functions/src/types.ts

```diff
@@ -55,11 +55,13 @@ export class Configs {
   constructor(
     input: KubernetesObject[] = [],
     functionConfig?: KubernetesObject,
-    results?: Result[]
+    results?: Result[],
+    expectedDataKeys?: string[]
```

Comment:
Can you please add comment for it?

Reply:
This should be passed into specific functions that need it (as an optional argument), not the ctor here.

Reply:
Yes, I talked to @runewake2 yesterday. And We agreed to change this back and not change the constructor.
We also want to ensure we handle this similarly in TS and Golang SDK.